patches/@uniswap+smart-order-router+2.10.0.patch

```diff
@@ -0,0 +1,738 @@
++++ b/node_modules/@uniswap/smart-order-router/build/main/providers/on-chain-quote-provider.js
+@@ -378,7 +378,8 @@ class OnChainQuoteProvider {
+                 // To work around this and avoid throwing errors when really we just couldn't get a quote, we catch this
+                 // case and return 0 quotes found.
+                 if ((this.chainId == util_1.ChainId.ARBITRUM_ONE ||
+-                    this.chainId == util_1.ChainId.ARBITRUM_RINKEBY) &&
++                    this.chainId == util_1.ChainId.ARBITRUM_RINKEBY ||
++                    this.chainId == util_1.ChainId.ARBITRUM_GOERLI) &&
+                     lodash_1.default.every(failedQuoteStates, (failedQuoteState) => failedQuoteState.reason instanceof ProviderGasError) &&
+                     attemptNumber == this.retryOptions.retries) {
+                     log_1.log.error(`Failed to get quotes on Arbitrum due to provider gas error issue. Overriding error to return 0 quotes.`);
+diff --git a/node_modules/@uniswap/smart-order-router/build/main/providers/token-provider.d.ts b/node_modules/@uniswap/smart-order-router/build/main/providers/token-provider.d.ts
+index 9befa84..265af02 100644
+--- a/node_modules/@uniswap/smart-order-router/build/main/providers/token-provider.d.ts
++++ b/node_modules/@uniswap/smart-order-router/build/main/providers/token-provider.d.ts
+@@ -61,6 +61,7 @@ export declare const DAI_ARBITRUM_RINKEBY: Token;
+ export declare const USDT_ARBITRUM_RINKEBY: Token;
+ export declare const USDC_ARBITRUM_RINKEBY: Token;
+ export declare const UNI_ARBITRUM_RINKEBY: Token;
++export declare const USDC_ARBITRUM_GOERLI: Token;
+ export declare const WMATIC_POLYGON: Token;
+ export declare const WETH_POLYGON: Token;
+ export declare const USDC_POLYGON: Token;
+diff --git a/node_modules/@uniswap/smart-order-router/build/main/providers/token-provider.js b/node_modules/@uniswap/smart-order-router/build/main/providers/token-provider.js
+index a164918..6ca1834 100644
+--- a/node_modules/@uniswap/smart-order-router/build/main/providers/token-provider.js
++++ b/node_modules/@uniswap/smart-order-router/build/main/providers/token-provider.js
+@@ -56,6 +56,7 @@ exports.DAI_ARBITRUM_RINKEBY = new sdk_core_1.Token(util_1.ChainId.ARBITRUM_RINK
+ exports.USDT_ARBITRUM_RINKEBY = new sdk_core_1.Token(util_1.ChainId.ARBITRUM_RINKEBY, '0x920b9301c2de92186299cd2abc7199e25b9728b3', 6, 'UDST', 'Tether USD');
+ exports.USDC_ARBITRUM_RINKEBY = new sdk_core_1.Token(util_1.ChainId.ARBITRUM_RINKEBY, '0x09b98f8b2395d076514037ff7d39a091a536206c', 6, 'USDC', 'USD//C');
+ exports.UNI_ARBITRUM_RINKEBY = new sdk_core_1.Token(util_1.ChainId.ARBITRUM_RINKEBY, '0x049251a7175071316e089d0616d8b6aacd2c93b8', 18, 'UNI', 'Uni token');
++exports.USDC_ARBITRUM_GOERLI = new sdk_core_1.Token(util_1.ChainId.ARBITRUM_GOERLI, '0x8FB1E3fC51F3b789dED7557E680551d93Ea9d892', 6, 'USDC', 'USD//C'); // This is the AAVE Test USDC
```

Comment:
what is this line doing?

Reply:
This add the ArbGoerli version of USDC, the comment is incorrect and removed